### DISCUSSION TITLE
TeslaMate, Home Assistant, and known_devices

### DISCUSSION THREAD

**Discussion by m-theredhead** [teslamate-org/teslamate#1134] [Q&A]
**Describe the bug**

Installed teslamate and it seems to be working well.   I have weeks of data on trips, charging, location, etc.

Also added teslamate to Home Assistant.   In general, this has worked well as I see the state of the sensors and replicated the dashboard.

The only issue is the GPS location of the car not getting updated in Home Assistant.  The sensor 'tesla_location' doesn't seem to exist within home assistant.

I tried creating this in the known_devices.yaml, but it seems like this is being ignored.   Looking online, it seems to indicate this file has been deprecated, but it is not clear what should be used instead.

I do see the sensor.tesla_latitude and sensor.tesla_longitude values are getting set and it feels like creation of the tesla_location is missing.

Any guidance on how to get this to work?

- TeslaMate version: 1.20.1
- Type of installation: Docker
- OS TeslaMate is installed on: Debian Buster + Docker ce
- User OS & Browser: Fedora 33 / Firefox
- Home Assistant: 2020.12.1 Docker
- Others:

**Comment by Sjorsa** [comment]
Did you add [this ](https://docs.teslamate.org/docs/integrations/home_assistant/#automationyaml) automation?

**Comment by bstadnik1** [comment]
May I ask how you configured Home Assistant to talk to TeslaMate? I can't seem to figure it out. I installed the MQTT add-on and point Home Assistant to the IP address of my Raspberry Pi where Telsamate is installed, but it doesn't seem to do anything

**Reply by Sjorsa** [reply]
Did you follow this explanation? https://docs.teslamate.org/docs/integrations/home_assistant/ You need to add the sensors etc manually.

**Comment by Sjorsa** [comment]
Turns out the cause was I typed longitude as longtitude somewhere 🤦
